--- a/setup.py	2026-01-20 14:33:21.754343473 +0700
+++ b/setup.py	2026-01-20 14:44:21.582348333 +0700
@@ -266,10 +266,12 @@
     zip_safe=False,
     # Add jetson_variables in /opt/jetson_stats
     # http://docs.python.org/3.4/distutils/setupscript.html#installing-additional-files
-    data_files=[('jetson_stats', ['services/jtop.service', 'scripts/jtop_env.sh'])],
+    data_files=[
+        ('lib/systemd/system', ['services/jtop.service']),
+        ('etc/profile.d', ['scripts/jtop_env.sh']),
+    ],
     # Install extra scripts
-    cmdclass={'develop': JTOPDevelopCommand,
-              'install': JTOPInstallCommand},
+    cmdclass={'develop': JTOPDevelopCommand },
     # The following provide a command called `jtop`
     entry_points={'console_scripts': [
         'jtop=jtop.__main__:main',
